Src/Plugins/Tests.Security/Validators/SEC101_54.AlibabaAccessKeyValidatorTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    public class AlibabaAccessKeyValidatorTests : AlibabaAccessKeyValidator
```

Comment:
>AlibabaAccessKeyValidator [](start = 50, length = 25)

do we need to do this way? is it better than calling directly? something like alibabavalidator.isvalid.....?

Reply:
Fixed

---
In reply to: [591819729](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591819729) [](ancestors = 591819729)